easybuild/easyconfigs/g/GCC/GCC-system.eb

```diff
@@ -8,10 +8,13 @@
 
 easyblock = 'SystemCompiler'
 
-name = "GCC"
-version = "system"
+name = 'GCC'
+# using 'system' as a version instructs the SystemCompiler easyblock to derive the actual compiler version,
+# and define $EBVERSION* accordingly (yet generate a module named GCC/system);
+# if an actual version is specified here, it will be checked against the version of the system compiler that was found
+version = 'system'
 
-homepage = "http://gcc.gnu.org/"
+homepage = 'http://gcc.gnu.org/"
```

Comment:
Nice catch, sorry about that (but Jenkins would have noticed if you would have merged this ;-)).

Fixed!